eesp-ikev2.org

```diff
@@ -289,7 +289,28 @@ numbers are in use.  For a given Sub SA key, sequence numbers MUST
 remain unique and monotonically increasing to meet cryptographic
 requirements.
 
-** Negotiation of Sub SA
+** Negotiation of Sub SAs
+
+Sub SAs are negotiated during the creation of an EESP Child SA, such
+as in the IKE_AUTH or CREATE_CHILD_SA new SA exchanges. To propose
+Sub SA support, the initiator MUST include a USE_EESP_SUB_SA
+notification, which specifies the "Minimum Number of Sub SAs"
```

Comment:
Isn't it the maximum number this particular peer accepts as well?

Reply:
It should be maximum number. Each side indicates the upper limit for Sub SAs it can handle as a receiver. THere is no negotiation, just announcement. The actual number of Sub SAs may differ in different directions.